src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -279,40 +280,42 @@ bool AllowProtection(const ConnectionHandlerSettings& settings,
 }
 #endif  // ENABLE_SECURITY
 
-uint32_t ConnectionHandlerImpl::OnSessionStartedCallback(
+void ConnectionHandlerImpl::OnSessionStartedCallback(
```

Comment:
Done in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/9f0795170b9cc46c6c8565ba967da6ae3bdbc613.